src/__tests__/utils.ts

```diff
@@ -37,3 +37,14 @@ test('isEmpty', () => {
   expect(isEmpty(NaN)).toEqual(true);
   expect(isEmpty([''])).toEqual(false);
 });
+
+test('stringifyDefined', () => {
+  expect(stringifyDefined(null)).toEqual('null');
+  expect(stringifyDefined(undefined)).toEqual('undefined');
+  expect(stringifyDefined([])).toEqual('[]');
+  expect(stringifyDefined({})).toEqual('{}');
+  expect(stringifyDefined(1)).toEqual('1');
+  expect(stringifyDefined(false)).toEqual('false');
+  expect(stringifyDefined({ test: undefined, test2: undefined })).toEqual('{}');
+  expect(stringifyDefined({ test: 'value', test2: undefined })).toEqual('{"test": "value"}');
```

Comment:
nit: here I would use a test.each to easily display all combinations of input / expectedResult but that's personal preference

Reply:
I will consider this